databases/week3/databases_week3_mealsharing_DML.sql

```diff
@@ -0,0 +1,234 @@
+-- Quaries to write on meal table
+-- Get all meals
+SELECT
+  meal.title
+from
+  meal;
+-- Add a new meal
+INSERT INTO
+  meal(
+    `title`,
+    `description`,
+    `location`,
+    `when`,
+    `max_reservations`,
+    `price`,
+    `created_date`
+  )
+VALUES(
+    'dish of the day',
+    'the dish of the day is a choise of the main chef and is different efery day. Please ask for details',
+    'china town lyngby',
+    '2022-02-16 09:09:09',
+    200,
+    90,
+    '2022-02-15'
+  );
+-- Get a meal with any id, fx 1
+SELECT
+  -- meal.title
+  *
+FROM
+  meal
+WHERE
+  id = 1;
+-- Update a meal with any id, fx 1. Update any attribute fx the title or multiple attributes
+UPDATE
+  meal
+SET
+  meal.description = 'traditional spicy pizza'
+WHERE
+  meal.title LIKE '%pizza';
+-- Delete a meal with any id, fx 1
+DELETE FROM
+  meal
+WHERE
+  id = 1;
+-- Quries to write on Reservation table
+  -- Get all reservations
+select
+  meal.title as Food,
+  reservation.number_of_guests as 'Total Guests',
+  meal.when as 'Date of Event'
+from
+  reservation
+  JOIN meal on reservation.meal_id = meal.id;
```

Comment:
you only needed to get the reservations here, no need for the meal :)

Reply:
ok